Say Their Names/Source/Controller/Home/HomeController.swift

```diff
@@ -41,6 +49,11 @@ class HomeController: BaseViewController {
         // Select first location by default
         let selectedIndexPath = IndexPath(item: 0, section: 0)
         locationCollectionView.selectItem(at: selectedIndexPath, animated: false, scrollPosition: .centeredVertically)
+        
+        // hide the major parts of the homeview
+        // so the launch screen will appear
+        homeView.hideable.forEach { $0.isHidden = true }
```

Comment:
this shouldn't be needed anymore after latest fixes to launch screen code 🤔

Reply:
I'm willing to believe you. With the branch I have, it is needed. Can we merge this into development and then see?

Reply:
Fixed this during merge 🙂

Reply:
(This part of the code was moved to the tabbar subclass)